Updating the causes as risks section in impairments (#1447)
ihmeuw/vivarium_research: docs/source/model_design/vivarium_model_components/impairments/index.rst

Have Causes Act as "Risks"
++++++++++++++++++++++++++

.. note::
  

>   While we think this methodology will work well, it has not been tested in a 

Reviewer: I think we should note somewhere that this idea and the analogy with mortality only work well if the impairment has no remission.

  model up to this point. Therefore the implemenation remains theoretical, and 
  some details to the approach would need to be finalized before it could be 
  used. 

> Another option is to have cause models that act as "risks" for other 
> causes. In effect, we would adjust the incidence of the impairment 

Reviewer: ```suggestion
Another option is to have cause models that act as "risks" on the
impairment model. In effect, we would adjust the incidence of the impairment
```
